src/middleware/router.js

```diff
@@ -517,7 +518,11 @@ function sendRequest(ctx, route, options) {
   }
 
   const requestDelegate =
-    route.type === 'kafka' ? sendKafkaRequest : sendHttpRequest
+    route.type === 'kafka'
+      ? sendKafkaRequest
+      : route.type === 'rabbitmq'
+      ? sendRabbitMQRequest
+      : sendHttpRequest
```

Comment:
This chained logic is a bit confusing, perhaps rather split this out to a switch statement or something rather than chaining ternaries.